src/ui/PromotedResultsBadge/PromotedResultsBadge.ts

```diff
@@ -40,13 +40,13 @@ export class PromotedResultsBadge extends Component {
    */
   static options: IPromotedResultsBadgeOptions = {
     /**
-     * Specifies if a badge should be added to "Featured Results" configured through a [Coveo Query Pipeline](http://www.coveo.com/go?dest=cloudhelp&lcid=9&context=126).
+     * Specifies if a badge should be added to "Featured Results" configured through a [Coveo Query Pipeline](https://docs.coveo.com/en/1961/).
```

Comment:
As it currently stands, the inline link doesn't really make sense to me.

I think the docs should rather be:

```
/**
 * Whether to show a badge when a result was promoted through a query pipeline _featured results_ rule.
 * @externaldocs [Adding and Managing Query Pipeline Featured Results](https://docs.coveo.com/1961/)
 */
```